packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -381,7 +152,7 @@ class FormAds extends React.Component {
             ),
         ];
 
-        const amount_messages = field_name => [
+        const offer_amount_messages = field_name => [
```

Comment:
Since these are all functions we should camelCase them, and preferably prefix them with an action such as `get` e.g. `getOfferAmountMessages`